batch/niji_sample.ts

```
@@ -0,0 +1,184 @@
+    voiceId: id,
+  };
+  beats.push({
+    id: key,
+    speaker: key,
+    text: `こんにちは, これは音声テストです。音声は ${name} です`,
+    image: {
+      type: "markdown",
+      markdown: [`# Voice is ${name} / ${id}`],
+    },
+  });
+  console.log(name);
```

ESLint

[error] Unexpected console statement.

(no-console)